ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml

```diff
@@ -216,7 +216,7 @@ tests:
     workflow: openshift-qe-installer-aws-ovn-fips
   timeout: 5h0m0s
 - as: node-density-24nodes-ovn-fips
-  cron: 0 9 * * 4
+  cron: 0 1 2,9,16,23 * *
```

Comment:
@afcollins Same reason as above. Pls move this to always_run: false.

Reply:
done